src/rpc/blockchain.cpp

```diff
@@ -2406,6 +2463,7 @@ static const CRPCCommand commands[] =
     { "blockchain",         "getblockstats",          &getblockstats,          {"hash_or_height", "stats"} },
     { "blockchain",         "getbestblockhash",       &getbestblockhash,       {} },
     { "blockchain",         "getblockcount",          &getblockcount,          {} },
+    { "blockchain",         "getchainalgostats",      &getchainalgostats,      {} },
```

Comment:
Note; we'll have to re-order this to reflect the function order (was going to say put it above getbestblockhash or move the function to below getblockcount, but I see it already is.  So out of scope for this but a note to be aware of)

Reply:
I realized it after the submitted the PR. I will fix it with the copyright change.